src/test/java/org/prebid/server/bidder/orbidder/OrbidderBidderTest.java

```diff
@@ -1,41 +1,102 @@
 package org.prebid.server.bidder.orbidder;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
+import com.iab.openrtb.request.Banner;
 import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
 import com.iab.openrtb.response.BidResponse;
 import com.iab.openrtb.response.SeatBid;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
 import org.prebid.server.VertxTest;
 import org.prebid.server.bidder.model.BidderBid;
 import org.prebid.server.bidder.model.BidderError;
 import org.prebid.server.bidder.model.HttpCall;
 import org.prebid.server.bidder.model.HttpRequest;
 import org.prebid.server.bidder.model.HttpResponse;
 import org.prebid.server.bidder.model.Result;
+import org.prebid.server.currency.CurrencyConversionService;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.request.orbidder.ExtImpOrbidder;
 
+import java.math.BigDecimal;
 import java.util.Collections;
 import java.util.List;
+import java.util.function.UnaryOperator;
 
 import static java.util.Collections.emptyList;
 import static java.util.Collections.singletonList;
+import static java.util.function.UnaryOperator.identity;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.assertj.core.api.AssertionsForClassTypes.tuple;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.given;
 
 public class OrbidderBidderTest extends VertxTest {
 
     private static final String ENDPOINT_URL = "https://test.endpoint.com/";
 
     private OrbidderBidder orbidderBidder;
 
+    @Rule
+    public final MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Mock
+    private CurrencyConversionService currencyConversionService;
+
     @Before
     public void setUp() {
-        orbidderBidder = new OrbidderBidder(ENDPOINT_URL, jacksonMapper);
+        orbidderBidder = new OrbidderBidder(ENDPOINT_URL, currencyConversionService, jacksonMapper);
     }
```

Comment:
Not sure if it is better to use this approach?
```
private CurrencyConversionService currencyConversionService;

@Before
public void setUp() {
    currencyConversionService = Mockito.mock(CurrencyConversionService.class);
    orbidderBidder = new OrbidderBidder(ENDPOINT_URL, currencyConversionService, jacksonMapper);
}
```

Reply:
> Mockito.mock() method and @Mock annotation are doing slightly the same, which is defining a mock object in unit tests.

I think use `@Mock` annotation it's more project style :)